spec/core_spec.rb

```diff
@@ -12,22 +12,34 @@
     Standupguy::Core.new(subject)
   end
 
+  describe "defaults" do
+    subject{ Standupguy::Core.new({ }) }
```

Comment:
Space missing to the left of {.<br>Space inside empty hash literal braces detected.